installer.sh.in

```diff
@@ -1328,6 +1328,13 @@ ${BOLD}Do you want to continue?${RESET}" 20 80 || return
     # clean up polkit rule - it's only useful in live systems
     rm -f $TARGETDIR/etc/polkit-1/rules.d/void-live.rules
 
+    # remove agetty services for devices that are not present
+    for service in /var/service/agetty-*; do
+        if [ ! -e /dev/"$(cut -d'-' -f2 <<< "$service")" ]; then
```

Comment:
```
[ -e "/dev/${service#*agetty-}" ] || rm -v "$service"
```

But shoudln't this be done in `$TARGETDIR`?

Reply:
we should instead figure out why those services are getting auto-enabled in the first place, and then fix that

Reply:
This dracut hook enables all installed services on boot that are not excludes: https://github.com/void-linux/void-mklive/blob/master/dracut/vmklive/services.sh

Reply:
well, since this dracut hook includes `ttyS*` already, it would make sense to add `hvc0` and `hvsi0` in there as well

Reply:
actually, i have that already done here, along with proper enablement of serial console in live: https://github.com/void-ppc/void-mklive/commit/8a3c5c0316be4e95bac72232729bb9e6f9aa3ebc

Reply:
Great to see this fixed, albeit not the way I imagined.

How would the people that needs `hvc0`/`hvsi` consoles would enable them? Would that be specifying `console` on the command line (similarly as done for GCP images [here](https://github.com/void-ppc/void-mklive/blob/master/mkimage.sh.in#L309)), taken into account by dracut script of the live ISO ([here](https://github.com/void-linux/void-mklive/blob/master/dracut/vmklive/getty-serial.sh#L7))?

Reply:
yeah just add the `console=` you want, e.g. `console=hvc0` or `console=ttyS0` and the new dracut hook will pick it up and enable the service